mautic/mautic
# Drop PHP 7.2 support

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | features
| Bug fix?                               | n/a
| New feature?                           | n/a
| Deprecations?                          | n/a
| BC breaks?                             | n/a
| Automated tests included?              | n/a
| Related user documentation PR URL      | n/a
| Related developer documentation PR URL | n/a
| Issue(s) addressed                     | n/a




#### Description:

Part of #9593, this PR drops support for PHP 7.2 (targeted for Mautic 3.3.0).

PHP 7.2 [went EOL (End of Life) on November 30, 2020](https://www.php.net/supported-versions.php) and therefore isn't receiving security updates anymore. It is now safe to drop support for it, as Mautic 3.2 supports both PHP 7.3 and 7.4. It will also allow us to do things like [upgrading to PHPUnit 9](https://phpunit.de/supported-versions.html).

**We just need to decide whether we want to drop support in Mautic 3.2.1 already (scheduled for December 28, 2020) or for Mautic 3.3.0 (February 22, 2021)** @mautic/core-team 



#### Steps to test this PR:
1. Checkout this PR locally
2. Make sure you're running PHP 7.2
3. You should receive an error message that the PHP version is too low
